client/src/components/OverviewFolder/CarouselThumbnail.jsx

```diff
@@ -14,23 +15,30 @@ const CarouselThumbnail = (props) => {
     for (let i = index; i < index + 5; i++) {
       if (props.photos[i]) {
         array.push(props.photos[i]);
-      }
-    }
-    lastPhotoInCarousel = array[array.length - 1]
+      };
+    };
+    lastPhotoInCarousel = array[array.length - 1];
     return array;
-  }
+  };
+
+  let imgSelector = function (src) {
+    props.setMainPhoto(src);
+    setSelectedPhoto(src);
+  };
 
   return (
     <div className="horizontalCarousel" >
-      {index === 0 ? null : <button className='topThumbnailButton' type="button" onClick={() => {setIndex(index - 1)}}>▲</button>}
+      {index === 0 ? null : <button className='topThumbnailButton' type="button" onClick={() => { setIndex(index - 1); }}>▲</button>}
       <div id="carouselThumbnail">
-      {props.photos ? thumbnail().map(({thumbnail_url}, index) => (
-        <div key={index}>
-           <img className="column" src={thumbnail_url} alt="placeHolder"></img>
-        </div>
-      )) : null}
+        {thumbnail().map(({thumbnail_url}, index) => (
+          <div key={index}>
+            {thumbnail_url !== selectedPhoto ?
+              <img className="column" src={thumbnail_url} onClick={(e) => { imgSelector(e.target.src); }}></img> :
```

Comment:
can modify onClick={e => imgSelector(e)}
(makes it a little cleaner. see line 24 for func changes